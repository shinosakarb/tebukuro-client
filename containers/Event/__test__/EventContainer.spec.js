import React              from 'react'
import { mount, shallow } from 'enzyme'
import ToJson             from 'enzyme-to-json'
import { Event }          from '../index.js'
import Params             from '../../../factories/Event'

jest.mock('../../../components/Event', () => 'EventComponent')

describe('Event container', () => {
  it('should render self and subcomponents', () => {
    const wrapper = shallow(<Event event={ Params.event1 }/>)
    const tree = ToJson(wrapper)
    expect(tree).toMatchSnapshot()
  })
})
