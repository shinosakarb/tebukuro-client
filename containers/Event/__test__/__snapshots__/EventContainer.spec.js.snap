// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event container should render self and subcomponents 1`] = `
<div>
  This is the Event container.
  <EventComponent
    event={
      Object {
        "description": "This is the first event.",
        "id": 1,
        "name": "event1",
      }
    }
  />
</div>
`;
